app/helpers/project_controller_helper.rb

```diff
@@ -11,4 +11,24 @@ def project_tags_link_list(project, type)
       link_to tag.name, projects_path(tags: tag.name)
     end.join(', ').html_safe
   end
+  
+  def projects_tech_and_cause_tags_lists(projects_array)
```

Comment:
Assignment Branch Condition size for projects_tech_and_cause_tags_lists is too high. [15.26/15]